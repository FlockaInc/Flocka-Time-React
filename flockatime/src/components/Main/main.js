import React from 'react';
import './main.css';

import Jumbotron from '../Jumbotron/jumbotron';
import ChartDisplay from '../ChartDisplay/chartDisplay';
// import graph
// import table

function Main() {

  return (
    <div className='main'>
      <Jumbotron />
      <div className='container'>
        <ChartDisplay />
      </div>
    </div>
  )
}

export default Main;