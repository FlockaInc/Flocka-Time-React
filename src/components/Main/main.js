import React from 'react';
import './main.css';

import Jumbotron from '../Jumbotron/jumbotron';
import ChartDisplay from '../ChartDisplay/chartDisplay';
// import graph
import CoderTable from '../CoderTable/coderTable';

function Main() {

  return (
    <div className='main'>
      <Jumbotron />
      <div className='container'>
        <ChartDisplay />
        <CoderTable />
      </div>
    </div>
  )
}

export default Main;