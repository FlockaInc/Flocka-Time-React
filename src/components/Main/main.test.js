import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Main from './main';

afterEach(cleanup);

it('renders', () => {
  const { asFragment } = render(<Main />);
  expect(asFragment()).toMatchSnapshot();
});