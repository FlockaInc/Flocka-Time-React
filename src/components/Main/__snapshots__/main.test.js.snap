// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="main"
  >
    <div
      class="jumbotron"
    >
      <h1
        class="display-3 mx-5"
      >
        <strong>
          FLOCKA TIME
        </strong>
      </h1>
      <p
        class="lead text-white mx-5"
      >
        This is Flocka Time, an easy to use extension for Visual Studio Code that tracks you and your friends' coding time. Based off the Waka Time extension.
      </p>
    </div>
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-1"
        />
        <div
          class="col-10"
        />
        <div
          class="col-1"
        />
      </div>
      <table
        class="table table-hover table-sm text-white leaderboard my-3"
      >
        <thead>
          <tr>
            <th>
              Rank
            </th>
            <th>
              Programmer
            </th>
            <th>
              Hours Coded
            </th>
            <th>
              Daily Average
            </th>
          </tr>
        </thead>
        <tbody />
      </table>
    </div>
  </div>
</DocumentFragment>
`;
